.github/workflows/templates/test.yml

```diff
@@ -70,6 +70,7 @@ env:
                       -DBUILD_METAL=${BUILD_MD:-"ON"} \
                       -DBUILD_HPMC=${BUILD_HPMC:-"ON"} \
                       -DCUDA_ARCH_LIST="60;70" \
+                      -DENABLE_DEBUG_JIT=ON \
```

Comment:
This doesn't seem related to the PR description. Is there is a specific reason this was added here and other places in this PR?

Reply:
Changes to the CI configuration are not user-facing. Therefore, I do not document them in the change log.

I added this here so that I could see the error messages resulting from the failed NVRTC compilations in the CI tests. It shows up in the other files as those are generated from this template.